packages/generators/package.json

```diff
@@ -22,6 +22,7 @@
     "plugin-template"
   ],
   "dependencies": {
+    "@webpack-contrib/defaults": "^6.3.0",
```

Comment:
No need do this, we should reimplement logic and template here

Reply:
gotcha wip